internal/pkg/service/buffer/storage/statistics/cache/l1.go

```diff
@@ -83,7 +83,7 @@ func (c *L1) aggregateWithRev(_ context.Context, objectKey fmt.Stringer) (out st
 	c.cache.Atomic(func(t prefixtree.TreeReadOnly[statistics.Value]) {
 		for _, level := range storage.AllLevels() {
 			t.WalkPrefix(
-				c.repository.Schema().InLevel(level).InObject(objectKey).Prefix(),
+				c.repository.ObjectPrefix(level, objectKey),
 				func(_ string, v statistics.Value) bool {
```

Comment:
Este jeden drobny fix.

Ako sa snazim aby kazda domena "vlastnila" entity a schemu, tak by schema mala byt private.

Mimo `repository` ju potrebujeme iba v `cache`, ktora ma in-memory kopiu statistik z databazy 1:1.
Tak som na tento ucel pridal metodu `ObjectPrefix` a schemu som dal private.